kubejs/client_scripts/script.js

```diff
@@ -385,28 +385,28 @@ onEvent('item.tooltip', function (event) {
   }
 
   event.addAdvanced('thermal:lumium_glass', function (item, advanced, text) {
-    text.add(1, Text.red('Wither Resistant'));
+    text.add(1, Text.red('耐凋灵'));
   });
   event.addAdvanced('thermal:enderium_glass', function (item, advanced, text) {
-    text.add(1, Text.red('Wither Resistant'));
+    text.add(1, Text.red('耐凋灵'));
   });
   event.addAdvanced('astralsorcery:starmetal', function (item, advanced, text) {
-    text.add(1, Text.red('Wither Resistant'));
+    text.add(1, Text.red('耐凋灵'));
   });
   event.addAdvanced('patchouli:guide_book', function (item, advanced, text) {
     if (item.nbt['patchouli:book'] == 'patchouli:proletaria') {
-      text.add(1, Text.gold('Proletaria Information Pamphlet'));
+      text.add(1, Text.gold('Proletaria 信息小册子'));
```

Comment:
多少得正式点吧（虽然小册子这个翻译确实是Pamphlet的翻译之一）
```suggestion
      text.add(1, Text.gold('Proletaria 信息手册'));
```